firecracker-control/local.go

```diff
@@ -181,10 +197,25 @@ func (s *local) buildVMConfiguration(ctx context.Context, id string, req *proto.
 		driveBuilder = drivesBuilderFromProto(drive, driveBuilder, false)
 	}
 
-	// TODO: Reserve fake drives here (https://github.com/firecracker-microvm/firecracker-containerd/pull/154)
+	s.stubDriveIndex = len(driveBuilder.Build()) - 1
+	tempDir, err := ioutil.TempDir("", "stub-drives")
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create temp directory")
+	}
 
-	cfg.Drives = driveBuilder.Build()
+	s.cleanupFns = append(s.cleanupFns, func() error {
+		return os.RemoveAll(tempDir)
+	})
 
+	for i := 0; i < int(req.ContainerCount)-len(req.AdditionalDrives); i++ {
```

Comment:
The comment in firecracker.proto says that `ContainerCount` is `The number of dummy drives to reserve in advance before running FC instance`, so I would have thought this should just do `ContainerCount` iterations here, can you explain this more (potentially in a code comment too)?